src/test/scala/com/github/tminglei/slickpg/PgTsqlSupportSuite.scala

```diff
@@ -2,16 +2,17 @@ package com.github.tminglei.slickpg
 
 import java.util.UUID
 
-import org.scalatest.FunSuite
+import org.scalatest.funsuite.AnyFunSuite
 import slick.basic.StaticDatabaseConfig
 
 /*
  * NOTE: to check it, we need move `MyPostgresDriver.scala` from test folder to main folder
  */
 //@StaticDatabaseConfig("file:src/test/resources/application.conf#tsql")
-//class PgTsqlSupportSuite extends FunSuite {
-//
-//  test("tsql support - simple") {
+class PgTsqlSupportSuite extends AnyFunSuite {
+
+  test("tsql support - simple") {
+    pending
```

Comment:
This particular test suite was giving mer errors when compiled against `2.11.12` with an sbt exception. Uncommenting the test, and setting to pending resolved the compilation issue.